src/app/shell/workshop-details/workshop-page/workshop-page.component.html

```diff
@@ -16,10 +16,14 @@ <h1 class="title">{{workshop.title}}</h1>
         </div>
 
       </div>
-      <p fxLayout="row" fxLayoutAlign="start center"><img src="assets/icons/rate.png" alt="">{{workshop.rate}}</p>
-      <a [routerLink]="'./provider-about'" class="workshop-link">
-        <p fxLayout="row" fxLayoutAlign="start center"><img src="assets/icons/home.svg" alt="">Школа танців</p>
-      </a>
+      <p fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
+        <img src="assets/icons/rate.png" alt="">
+        <span>{{workshop.rating}}</span>
+      </p>
+      <p fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
+        <img src="assets/icons/home.svg" alt="">
+        <a [routerLink]="'./provider-about'" class="workshop-link">Школа танців</a>
+      </p>
```

Comment:
I guess you can also pass providerId to the  <app-provider-about [provider]=""></app-provider-about> and <app-all-provider-workshops [provider]=""></app-all-provider-workshops> below.  There we can distpatch getProviderById and  display real info about the provider. But it also could be a separated task